arangodb-net-standard/Serialization/IContentSerialization.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.IO;
+
+namespace ArangoDBNetStandard.Serialization
+{
+    public interface IContentSerialization
```

Comment:
Actually I think I would prefer using the same naming scheme as for the transport layer, e.g. `IApiClientSerialization`.

@Actify-Inc/arangodb-net-standard-devs any preference?

Reply:
Yes agreed.